shell/src/dapps/o-marketlisting/pages/MarketlistingDetail.svelte

```diff
@@ -76,7 +76,7 @@ onMount(async () => {
     shopOwner = shopOwnerData.contactAddress_Profile.members;
     const currentDateIndex = new Date().getDay();
 
-    isMyShop = $me.circlesAddress === shopOwner[0].circlesAddress;
+    isMyShop = $me.circlesAddress === business.circlesAddress || $me.circlesAddress === shopOwner[0].circlesAddress;
```

Comment:
what if shopOwner is undefined? then this line will still break right?

Reply:
@codeho the only case when shopOwner is undefined is when I am in the context of that shop, and in that case business.circlesAddress will be true

In the expression true || undefined, if the first operand,is true, is truthy. Since the || operator stops evaluation as soon as it finds a truthy operand, the result of this expression will be true.

Reply:
alright then. I'd love if you'd type let shopOwner; even though it's not part of this change :)